Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -223,15 +254,22 @@ def main():
         print(f'{t.cyan}Updated base branch of PR "{pr_number}" to "{new_branch_name}"{t.normal}')
 
     # assign reviewers / request review from
-    reviewer_to_assign = determine_reviewer(REVIEWERS, content_repo)
-    pr.add_to_assignees(reviewer_to_assign)
-    pr.create_review_request(reviewers=[reviewer_to_assign])
-    print(f'{t.cyan}Assigned user "{reviewer_to_assign}" to the PR{t.normal}')
-    print(f'{t.cyan}Requested review from user "{reviewer_to_assign}"{t.normal}')
+    content_reviewer = determine_reviewer(REVIEWERS, content_repo)
+    pr.add_to_assignees(content_reviewer)
+    reviewers = [content_reviewer]
+
+    # Add security reviewer if the PR contains
+    if is_requires_security_reviewer(pr_files):
+        reviewers.append(SECURITY_REVIEWER)
+        pr.add_to_assignees(SECURITY_REVIEWER)
+        pr.add_to_labels(SECURITY_LABEL)
+
+    pr.create_review_request(reviewers=reviewers)
+    print(f'{t.cyan}Assigned and requested review from "{",".join(reviewers)}" to the PR{t.normal}')
```

Comment:
I tried that but had to [revert](https://github.com/demisto/content/pull/28109/commits/06d18d4b11b6e93363f98762840de4296b057d6a) it failed because `add_to_assignees` failed getting an arg of type `List[str]`, It's expecting either a `List[github.NamedUser.NamedUser]` (https://pygithub.readthedocs.io/en/latest/github_objects/NamedUser.html#github.NamedUser.NamedUser). or an `str`.